subscriptions-manager/src/main/java/io/arlas/subscriptions/service/UserSubscriptionManagerService.java

```diff
@@ -73,24 +69,22 @@ public UserSubscription postUserSubscription(UserSubscription userSubscription,
     }
 
     public Optional<UserSubscription> getUserSubscription(String id, Optional<String> user, boolean deleted) throws ArlasSubscriptionsException {
-        logger.debug(String.format("User %s requests subscription %s (deleted %b)", user.orElse(UNKNOWN_USER), id, deleted));
         return this.daoDatabase.getSubscription(id, user, deleted);
     }
 
     public void deleteUserSubscription(UserSubscription userSubscription) throws ArlasSubscriptionsException {
-        logger.debug(String.format("User %s deletes subscription %s", userSubscription.created_by, userSubscription.getId()));
         this.daoDatabase.setUserSubscriptionDeletedFlag(userSubscription, true);
 
         try {
             this.daoIndexDatabase.setUserSubscriptionDeletedFlag(userSubscription, true);
```

Comment:
Doesn't a setter return an updated instance of the UserSubscription (it is strange to update it manually)?

Reply:
Moving it to dao as for the other updates.